pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1066,14 +1067,16 @@ def test_dti_iadd_int(self, tz_naive_fixture, one):
                             periods=10, tz=tz)
         expected = pd.date_range('2000-01-01 10:00', freq='H',
                                  periods=10, tz=tz)
-        rng += one
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
```

Comment:
why can't you check the stacklevel anymore?

Reply:
We basically never did.  I never got the stacklevel checks to pass in any consistent way.